sqs/src/test/scala/uk/gov/nationalarchives/DASQSClientTest.scala

```diff
@@ -36,20 +39,23 @@ class DASQSClientTest extends AnyFlatSpec with MockitoSugar {
     val mockResponse = CompletableFuture.completedFuture(SendMessageResponse.builder().build())
     when(sqsAsyncClient.sendMessage(sendMessageCaptor.capture())).thenReturn(mockResponse)
 
-    val client = new DASQSClient[IO](sqsAsyncClient)
-    client.sendMessage("https://test")(Test("testMessage", "testValue")).unsafeRunSync()
+    val fifoConfig = FifoQueueConfiguration("MessageGroupId", "MessageDeduplicationId")
+    val client = DASQSClient[IO](sqsAsyncClient)
+    client.sendMessage("https://test")(Test("testMessage", "testValue"), Option(fifoConfig)).unsafeRunSync()
```

Comment:
Should we add a test or add table (and do a `forAll`) for the case where a `fifoConfig` was not passed in?

Reply:
Good point. I've added another test in, using forAll seems a bit much.